Umbraco.ModelsBuilder/Configuration/Config.cs

```diff
@@ -63,6 +64,10 @@ private Config()
             ModelsDirectory = HostingEnvironment.IsHosted
                 ? HostingEnvironment.MapPath(DefaultModelsDirectory)
                 : DefaultModelsDirectory.TrimStart("~/");
+            BinDirectory = HostingEnvironment.IsHosted
+                ? HostingEnvironment.MapPath(DefaultBinDirectory)
+                : DefaultModelsDirectory.TrimStart("~/");
```

Comment:
Shouldn't this be `DefaultBinDirectory.TrimStart("~/")` ?

Reply:
It should! This is an error by copying the ModelsDirectory.